defaults/manager_persistent_snapshots.yaml

```diff
@@ -3,78 +3,51 @@ aws:
   confirmation_stress_template: "cassandra-stress read cl=QUORUM n={num_of_rows} -schema 'keyspace={keyspace_name} replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq={sequence_start}..{sequence_end}"
   snapshots_sizes:
     5:
-      number_of_rows: 5242880
+      number_of_rows: 5368800
```

Comment:
just wondering

why the row number differ ? 
the calculation of data size wasn't accurate enough ?  i

Reply:
Sorry, my bad, I've copied it from somewhere without double-checking - the previous calculation is correct, need to fix it